spotify_db/Query.sql

```diff
@@ -0,0 +1,169 @@
+-- Top popular tracks
+SELECT t.track_name, ar.artist_name, al.album_name, t.popularity
+FROM tracks t
+JOIN artists ar ON t.artist_id = ar.artist_id
+JOIN albums al ON t.album_id = al.album_id
+WHERE t.popularity > (SELECT AVG(popularity) FROM tracks)
+ORDER BY t.popularity DESC; 
```

Comment:
This query looks good to me. 

One opportunity to expand this more is to have a boolean flag that identifies which track is above or below the average.

Sudo code below:

```
SELECT t.track_name, ar.artist_name, al.album_name, t.popularity, t.popularity > avg_populartiy AS is_popularity_above_avg
```

Tips: Use `is_` prefix for boolean column

Reply:
Having a full list of tracks below or under popularity average can help us potentially understand what makes a track popular, in comparison to tracks that are not as popular